frontend/src/scenes/session-recordings/saved-playlists/savedSessionRecordingPlaylistModelLogic.ts

```diff
@@ -8,7 +8,7 @@ import { deleteWithUndo, toParams } from 'lib/utils'
 import { DEFAULT_RECORDING_FILTERS } from 'scenes/session-recordings/playlist/sessionRecordingsListLogic'
 import { router } from 'kea-router'
 import { urls } from 'scenes/urls'
-import { openBillingPopupModal } from 'scenes/billing/v2/BillingPopup'
+import { openBillingPopupModal } from 'scenes/billing/v2/control/BillingPopup'
```

Comment:
For the popup, same as `BillingLocked`, it's just a wrapper.

In this case though it's worth considering if we want to have new popups, not just a wrapper.

Reply:
I removed it from the test because it's just a wrapper. If we end up wanting to change them we can move them into the test later.